contrib/upgrade_testnet/submit_upgrade_proposal.sh

```diff
@@ -63,7 +63,7 @@ sleep 6s
 
 echo ""
 echo "===> Getting proposal id"
-PROPOSAL_ID=$(desmos q tx $TX_HASH --output json 2>&1 | jq .logs[0].events[2].attributes[0].value -r)
+PROPOSAL_ID=$(desmos q tx $TX_HASH --output json 2>&1 | jq .logs[0].events[4].attributes[0].value -r)
```

Comment:
This is needed for Cosmos `v0.44` and later since they moved how events  are ordered inside a transaction result